src/darwin/Framework/CHIP/CHIPDeviceController.mm

```diff
@@ -359,6 +372,60 @@ - (void)registerCallbacks:appCallbackQueue onMessage:(ControllerOnMessageBlock)o
     self.onErrorHandler = onError;
 }
 
+// TODO Get this code one level up, so it is easily accessible from the application
+static void onBleError(chip::DeviceController::ChipDeviceController * deviceController, BLE_ERROR err)
```

Comment:
Can we file an issue to move this down into "libchip" not just in the Darwin framework?

Reply:
Actually those methods are called by libchip. These are arguments to ChipDeviceController.InitBle so the platform already add those callbacks.
The comment is about exposing them to the application layer instead of only having them into the framework layer.

For the record those are different signatures than the pre-existing callbacks for the transport layer. Which is something I don't particularly like and I would like to clean up.